packages/visual-group/src/encoder/cartesian-encoder.js

```diff
@@ -121,9 +121,18 @@ export default class CartesianEncoder extends VisualEncoder {
                     fieldArr.forEach((field, axisIndex) => {
                         const key = !axisType ? `0${cIdx}${axisIndex}` : `${rIdx}0${axisIndex}`;
                         const dom = unitDomains[`${rIdx}${cIdx}`];
+                        const typeOfField = field.subtype();
+                        const sortingDetails = field.data()._sortingDetails;
```

Comment:
@adarshlilha Add an api in datamodel to get the sorting details

Reply:
Added a method to fetch sortingDetails from the current field instance